internal/resources/planner.go

```diff
@@ -310,7 +310,7 @@ func (sp *sharePlanner) update() (changed bool, err error) {
 			changed = true
 		}
 	}
-	return
+	return changed, nil
```

Comment:
Is this needed? The function uses named return values [1]. Did you see a bug with this line?

1 -  https://go.dev/tour/basics/7

Reply:
I wasn't aware of named return values and to a person used to looking at C code, seemed wrong. I was just investigating the code around when I was confused about this. I've deleted the change but we should consider avoiding this particular feature to avoid such confusion. :)